src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Elasticsearch/RequestWrapper.cs

```diff
@@ -27,10 +30,22 @@ public CanWrapResponse CanWrap(InstrumentedMethodInfo methodInfo)
 
         public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall instrumentedMethodCall, IAgent agent, ITransaction transaction)
         {
-            //Elasticsearch.Net.HttpMethod,System.String,Elasticsearch.Net.PostData,Elasticsearch.Net.IRequestParameters
+            var indexOfRequestParams = 3; // unless it's Elasticsearch.Net/NEST and async
+
+            if (instrumentedMethodCall.IsAsync) 
```

Comment:
I think we might want to do some bounds checking, just in case they make some drastic change in a future release, but that can be done later.